src/pages/Browse/index.tsx

```diff
@@ -61,6 +64,19 @@ export class BrowseBase extends React.Component<Props> {
     );
   }
 
+  onSelectFile = (path: string) => {
+    const { _fetchVersionFile, dispatch, match } = this.props;
+    const { addonId, versionId } = match.params;
+
+    dispatch(
+      _fetchVersionFile({
```

Comment:
We don't seem to have a test for this. We do have a test that *something* is dispatched (using `fakeThunk` in the test), but we don't seem to have a test that verifies that `addonId`, `versionId` and `path` are passed into the dispatch.

Reply:
Indeed, there is no way to test that yet. I will file an issue.

Reply:
See: #262